test/infrastructure/kind/mapper.go

```diff
@@ -79,6 +79,28 @@ type Mapping struct {
 var preBuiltMappings = []Mapping{
 
 	// TODO: Add pre-built images for newer Kind versions on top
+	// Pre-built images for Kind v1.27.
+	{
+		KubernetesVersion: semver.MustParse("1.32.2"),
+		Mode:              Mode0_20,
+		Image:             "kindest/node:v1.32.2@sha256:f226345927d7e348497136874b6d207e0b32cc52154ad8323129352923a3142f",
+	},
+	{
+		KubernetesVersion: semver.MustParse("1.31.6"),
+		Mode:              Mode0_20,
+		Image:             "kindest/node:v1.30.6@sha256:28b7cbb993dfe093c76641a0c95807637213c9109b761f1d422c2400e22b8e87",
```

Comment:
```suggestion
		Image:             "kindest/node:v1.31.6@sha256:28b7cbb993dfe093c76641a0c95807637213c9109b761f1d422c2400e22b8e87",
```
This was a typo in kind release notes